management_api_app/db/events.py

```diff
@@ -10,14 +11,56 @@ async def connect_to_db(app: FastAPI) -> None:
     logging.debug(f"Connecting to {config.STATE_STORE_ENDPOINT}")
 
     try:
-        cosmos_client = CosmosClient(config.STATE_STORE_ENDPOINT, config.STATE_STORE_KEY)
+        cosmos_client = CosmosClient(config.STATE_STORE_ENDPOINT, config.STATE_STORE_KEY, connection_verify=False)
         app.state.cosmos_client = cosmos_client
         logging.debug("Connection established")
     except Exception as e:
         logging.debug(f"Connection to state store could not be established: {e}")
 
+# Bootstrapping is temporary while the API does not have a register spec api implemented.
```

Comment:
If this is only for adding samples for testing purposes - there is a script in the scripts folder that we can use for this.

IMHO we should not add samples as part of startup. 

We should however bootstrap the DB on startup with the base bundle specs, but from disc.

Reply:
Umm it's not testing purpose. This is a fixed entry till we have the authoring experience designed and done.

Reply:
Could load it from disk but this is clear what is being loaded instead of reading it from another place. Could move the json(dict) to another file if that makes it more elegant. Writing a json file to read into a dict is not needed for such a small case. If the dict goes larger then yes could put it into a file but don't think that will happen.

Reply:
So are you saying that this fixed entry will be here later too, but look different? or is it only here now, to be able to develop and test the other code? or is this the long term bootstrapping of the base bundle?

Reply:
While the authoring experience is undecided we can use this fixed entry to code POST /api/workspaces and use this fixed entry for bundle_url(let's change it when a real bundle exists on ACR). This will get removed as soon as we add another API POST /api/bundlespecs and make a wrapper on top of porter publish to also add bundle.json content to the store using the API.

Reply:
Hey, my 5c: I would also split this out into a sample provisioning, just to keep this clean of any sample code. So essentially when mare runs push_bundle, we can fire off a sample python that adds these.